virtcontainers/acrn.go

```diff
@@ -737,20 +743,14 @@ func (a *Acrn) GetACRNUUIDBytes(uid string) (uuid.UUID, error) {
 
 // GetNextAvailableUUID returns next available UUID VM creation
 // If no validl UUIDs are available it returns err.
-func (a *Acrn) GetNextAvailableUUID(uuidstore *store.VCStore) (string, error) {
+func (a *Acrn) GetNextAvailableUUID() (string, error) {
 	var MaxVMSupported uint8
 	var Idx uint8
 	var uuidStr string
 	var err error
 
-	if uuidstore == nil {
-		uuidstore = a.store
-	}
-
-	if uuidstore != nil { //use old store
-		if err = uuidstore.Load(store.UUID, &a.info); err != nil {
-			a.Logger().Infof("Load UUID store failed")
-		}
+	if err = a.loadInfo(); err != nil {
+		a.Logger().Infof("Load UUID store failed")
```

Comment:
- Probably worth logging the error here (may not be `ENOENT` but something else?
- I see the logic is similar to what was there before, but it might be worth adding a comment explaining why this is not an error scenario (`err` is not being returned).

Reply:
@jodh-intel I don't know why this didn't throw an error out, guess this is a bug? It's better to let ACRN guys take a look. I think it should return the error.

Reply:
@vijaydhanraj Do you think we should return the error here? I guess this is a bug but not sure.

Reply:
@WeiZhang555 , The intent here is to log a message when UUID.json file is not available. This is not a bug, as we proceed and create UUID.json file to store UUID availability.